src/js/store/configureStore.ts

```diff
@@ -34,6 +34,7 @@ export default function configureStore() {
     storageMiddleware,
   ];
 
+  /* istanbul ignore next */
```

Comment:
Might not be related to this PR - but shall we remove this block completely (along with the `isDev` variable)? I guess we can also remove `redux-logger` from `package.json`.

Reply:
Awesome! Even better 😎, I'll make this change.

I didn't want to overstep and really tried to avoid changing non-test files, but if you'd be okay with refactors, I'd be happy to go deeper and propose some cleanup changes.